src/System.Net.Http/src/System/Net/Http/Managed/HttpProxyConnectionHandler.Windows.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http
+{
+    internal sealed partial class HttpProxyConnectionHandler : HttpMessageHandler
+    {
+        static private IWebProxy GetDefaultroxy()
+        {
+            // Return Environmental proxy now for testing
+            // Windows normally does not use environment but it has system
+            // configuration. That has not been implemented yet.
+            // TODO #23150: windows portion
+            new HttpEnvironmentProxy();
```

Comment:
We should probably just return null for now.  That's effectively what we're doing today, right?

Reply:
it seems like when http_proxy was set, it would also be used on windows. 
I'd be happy to remove it but I thought it would make it easier to test proxies on Windows.

Reply:
Is http_proxy ever used on Windows?  I thought it was a Linux thing.

Reply:
normally it would not. But existing Managed/HttpProxyConnectionHandler.cs is not OS specific as far as I can tell. As I said I'm not sure if that is useful right now for anybody. I can add proper windows handling in next round.

Reply:
I'd implemented it to be used on Windows as well, for consistency across platforms, but I'm fine dropping that.